packages/edit-site/src/components/navigation-sidebar/navigation-panel/template-hierarchy.js

```diff
@@ -0,0 +1,16 @@
+const TEMPLATE_OVERRIDES = {
+	singular: [ 'single', 'page' ],
+	index: [ 'archive', '404', 'search', 'singular' ],
```

Comment:
I suppose 'home' would go in this list too since it overrides index. 

We should probably also add another for `home: [ 'front-page' ]` to determine whether or not it is superseded in the case that the site is set to show posts.

I also wonder how far we should extend these overrides.  Is there any use to adding keys for 'single', and 'archive' and corresponding arrays of the secondary templates that could override them?

Reply:
Thanks for catching those!

> I also wonder how far we should extend these overrides in general. Is there any use to adding keys for 'single', and 'archive' and corresponding arrays of the secondary templates that could override them?

I would avoid them since we can't predict accurately whether they are unused or not. `archive` can be overridden by `archive-$posttype`. Same for `single`, it can be overridden by `single-$posttype`.